ArticleTemplates/assets/scss/modules/_atoms.scss

```diff
@@ -4,3 +4,12 @@
     border: 0;
     margin: 1em 0;
 }
+
+
+iframe.scrolly {
+		position: sticky;
+		top: 0;
+		height: 100vh;
+		height: 100dvh;
```

Comment:
I am not familiar with CSS rule.  May I ask what we set the `height` attribute twice with different values here?  Is it to support old version of browser (webview) which may not support one of them?

Reply:
Yes, exactly – CSS is fault-tolerant by default so if the browser [does not support](https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic) the [dynamic viewport units](https://developer.mozilla.org/en-US/docs/Web/CSS/length#dynamic), it will ignore that rule, and fall back to the previous `height` rule that it can parse, which is the non-dynamic version `100vh`.